Gemfile

```diff
@@ -3,9 +3,12 @@ source 'https://rubygems.org'
 # 64bit Amazon Linux 2016.09 v2.3.0 running Ruby 2.3 (Puma)
 ruby '2.3.1'
 
-gem 'puma'
-
+gem 'activerecord'
+gem 'activerecord-postgis-adapter'
```

Comment:
At least https://github.com/rgeo/activerecord-postgis-adapter/search?utf8=%E2%9C%93&q=structure doesn't have any hits so maybe it's no problem now?